Octokit.Tests.Conventions/Octokit.Tests.Conventions.xproj

```diff
@@ -6,11 +6,11 @@
   </PropertyGroup>
   <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
-    <ProjectGuid>2740af44-cb2e-4fd8-a221-248693978a0c</ProjectGuid>
-    <RootNamespace>Octokit.Next.Tests</RootNamespace>
+    <ProjectGuid>a04124dc-d057-4db0-8e5f-ecd114fa371a</ProjectGuid>
+    <RootNamespace>OctokitBidon</RootNamespace>
```

Comment:
sorry, some hacky stuff I must have done at the beginning and forgot to remove them.